Annotation/Service.php

```diff
@@ -53,4 +53,10 @@
 
     /** @var array<string> */
     public $environments = array();
+
+    /** @var boolean */
+    public $autowire;
+
+    /** @var array<string> */
+    public $autowiring_types = array();
```

Comment:
The point was to keep the snake_case syntax for annotations' attributes. Just like `decoration_inner_name` (https://github.com/schmittjoh/JMSDiExtraBundle/pull/254/files#diff-2c2011597fae2a64c228c5d101095beaL49)

Which one should it be? Switch both to camelCase or keep snake_case for both?